src/main.rs

```diff
@@ -306,6 +306,17 @@ async fn run_command(cmd: &config::Command, hook: &Hook, request: &Request, body
 		set_request_environment(&mut command, request, None, remote_addr);
 	}
 
+	// Set hook environment variables
+	if let Some(environment) = &hook.environment {
+		for e in environment.iter() {
+			let parts: Vec<&str> = e.splitn(2, '=').collect();
+			if let Some(name) = parts.get(0) {
+				let value = parts.get(1).map(|v| v.trim()).unwrap_or("");
+	        	command.env(name, value);
+			}
+ 	   }
+	}
```

Comment:
```suggestion
	for (name, value) in &hook.environment {
		command.env(name, value);
	}
```

If we use an IndexMap we can simplify to this.